### ISSUE TITLE
[Feature Request] show Accessibility checks visually

### ISSUE THREAD

**Issue by yschimke** [takahirom/roborazzi#568]
Something similar to https://developer.android.com/guide/topics/ui/accessibility/testing#accessibility-scanner

Perhaps on failing accessibility checks, overlay the screenshot with the bounding box of the failure

https://developer.android.com/guide/topics/ui/accessibility/testing#accessibility-scanner

![image](https://github.com/user-attachments/assets/66572982-a115-4d43-89b9-8d02a984c07c)

**Comment by yschimke** [comment]
example

```
    // Rethrow the Accessibility exception once screenshots have passed
    if (accessibilityException != null) {
        accessibilityException.results.forEachIndexed { index, check ->
            val viewImage = check.viewImage
            if (viewImage is BitmapImage) {
                FileOutputStream(
                    "build/outputs/roborazzi/" +
                        "${screenshotName}_${deviceName}_$index.png",
                ).use {
                    viewImage.bitmap.compress(PNG, 100, it)
                }
            }
        }

        throw accessibilityException
    }
```

**Comment by takahirom** [comment]
I think we can use AwtRoboCanvas to save images while respecting user settings, such as image extensions, and to write text and frames.
